pkg/compact/downsample/streamed_block_writer.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/prometheus/tsdb"
 	"github.com/prometheus/tsdb/chunks"
 	"github.com/prometheus/tsdb/fileutil"
+	tsdb_errors "github.com/prometheus/tsdb/errors"
```

Comment:
```suggestion
	tsdberrors "github.com/prometheus/tsdb/errors"
```

Could we not use names with underscores please, as per https://talks.golang.org/2014/names.slide#5

Reply:
I used what tsdb does https://github.com/prometheus/tsdb/blob/19d402d154606a6d3c882420e26b9353b0f39a7c/repair.go#L26

Reply:
It doesn't mean that tsdb is doing it correctly :D 

Take a look at go stdlib, https://golang.org/pkg/ there are a bunch of packages that contain multiple words, which are squashed together `httputil`, `httptest`,...

Reply:
`_` in package name is Go antipatern, even though it's used in several places. See: https://blog.golang.org/package-names#TOC_2

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns, such as:

Just `multierrors` or `tsdberrors` is my vote